### PR TITLE
[AL-2005] Sanitized output going to Pantheon dashboard

### PR DESCRIPTION
**Pull request by TeslaDethray**

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@TeslaDethray I've fixed Travis for you.

**Comment by greg-1-anderson** [comment]
This all looks good, and seems like it should work today (although I did not test). The one concern I have is that this code is going to run all the way back to PHP 5.3 on the dashboard. Is PHP 7.2 the oldest PHP we can test on? These changes look good for PHP 5.3, but removing the older PHP versions in Travis leaves us open to future changes breaking older PHP. Maybe we should switch the tests to CircleCI, so that we detect such problems before we deploy?

Could potentially be follow-on work, but we should at least have a JIRA card before merging here.

**Comment by danielbachhuber** [comment]
@greg-1-anderson I've restored the PHP 5.6 build — PHP 5.6 was the oldest version previously.

It looks like the build fails on PHP 5.6 though:

image

image

Related #91

I've restarted the 5.6 build to see if it's just an intermittent failure.

File: php/pantheon/checks/cron.php [hunk 1]
```diff
@@ -49,6 +49,7 @@ public function checkCron() {
 		$total = 0;
 		$invalid = 0;
 		$overdue = 0;
+		$past = 0;
```
Comment (TeslaDethray) [inline php/pantheon/checks/cron.php]: This was for PHP 5.6 compat but I think it should be initialized anyhow.

**Comment by TeslaDethray** [comment]
@danielbachhuber @ataylorme @greg-1-anderson (Sorry if I @'d you unnecessarily) I can't get the PHP 5.6 tests to pass but in #95 I can show that it's not my changes that are causing it. Because we stopped deploying 5.6 to WP in the long, long ago I'm going to remove Travis' PHP 5.6 testing.
https://pantheon.io/blog/php-72-everywhere

**Comment by greg-1-anderson** [comment]
We still have a lot of WordPress sites running on PHP 5.x.  Many sites have not upgraded to the version of WordPress where the PHP version was set to PHP 7+ in pantheon.upstream.yml, and some of the WordPress sites that did upgrade explicitly downgraded PHP in their pantheon.yml due to incompatible plugins etc.

I created a JIRA card for follow-on work here: https://getpantheon.atlassian.net/browse/BUGS-2787

**Pull request merged**